src/api/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -479,6 +483,42 @@ else if (damageReductionAmount <= ArmorMaterial.DIAMOND.getDamageReductionAmount
         return 5;
     }
 
+    /**
+     * Estimates the fishing rod tier from available durability and enchantment status.
+     *
+     * @param itemStack the tool to check.
+     * @return equivalent tool level.
+     */
+    private static int getFishingRodLevel(final ItemStack itemStack)
+    {
+        if (itemStack.getItem() == Items.FISHING_ROD)
+        {
+            return (1 + getMaxEnchantmentLevel(itemStack));
+        }
+        if (!itemStack.isDamageable())
+        {
+            return 1;
+        }
+        /**
+         *  Because fishing rods don't follow ItemTier values, these numbers require an offset.
+         *  Wood + 6 is required to allow T1 huts to use up to vanilla fishing rods equivalents.
+         *  Wood + 22 allows T1 huts to use up to ThermalFoundation Silver fishing rods.
+         *  Iron + 6 allows T2 huts to use up to ThermalFoundation Iron fishing rods or Aquaculture Iron,
+         *  while still requiring T3 for ThermalFoundation advanced alloys, Aquaculture Neptunium,
+         *  or Aquaculture diamond rods.
+         * */
+        final int rodDurability = itemStack.getMaxDamage();
+        if (rodDurability <= (ItemTier.WOOD.getMaxUses() + 22))
```

Comment:
These hardcored constants should be configurable from the config

Reply:
Ok.  Added them to the server-specific config as fishingRodDurabilityAdjustT1 and fishingRodDurabilityAdjustT2.  Hopefully MinecoloniesAPIProxy.getInstance().getConfig().getServer() is a reasonable way to get to that; I'm not sure the performance ramifications of the different access methods, but didn't break across a few attempts.

Reply:
This is fine.